modules/lib/api/dts.xql

```diff
@@ -4,7 +4,7 @@ module namespace dts="http://teipublisher.com/api/dts";
 
 import module namespace config="http://www.tei-c.org/tei-simple/config" at "../../config.xqm";
 import module namespace http = "http://expath.org/ns/http-client";
-import module namespace router="http://e-editiones.org/roaster";
+import module namespace roaster"http://e-editiones.org/roaster";
```

Comment:
Huh, where's the `=`? Is it a display glitch again?

Reply:
@line-o Hmm, your commit id here is labeled by GitHub as "outdated" - perhaps that's why? The current one is https://github.com/eeditiones/tei-publisher-app/pull/43/files#diff-32914f99ffe435b058abce47f60ce9ee056448aa81e4ac65ab02155c2bd9cbccR5 and shows no syntax problem in the module import statement.